src/core/thread/mesh_forwarder.cpp

```diff
@@ -1364,11 +1364,22 @@ void MeshForwarder::HandleLowpanHC(uint8_t *               aFrame,
                                    const Mac::Address &    aMacDest,
                                    const otThreadLinkInfo &aLinkInfo)
 {
-    ThreadNetif &netif = GetNetif();
-    otError      error = OT_ERROR_NONE;
-    Message *    message;
+    ThreadNetif &netif   = GetNetif();
+    otError      error   = OT_ERROR_NONE;
+    Message *    message = NULL;
```

Comment:
Just curious why setting the `message` to `NULL`? It's fine.
Looking at code I think it does not matter so just checking if I am missing something.

Reply:
If the IPv6 header fails to parse in the newly added code, it goes to `exit:` with an uninitialized `message` pointer.

Reply:
Indeed! Important to have it... Thanks